src/Microsoft.Data.SqlClient/tests/ManualTests/AlwaysEncrypted/SqlBulkCopyTruncation.cs

```diff
@@ -0,0 +1,522 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Security.Cryptography.X509Certificates;
+using Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted.Setup;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.ManualTesting.Tests.AlwaysEncrypted
+{
+    [PlatformSpecific(TestPlatforms.Windows)]
+    public class SqlBulkCopyTruncation : IClassFixture<SQLSetupStrategyCertStoreProvider>, IDisposable
+    {
+        private SQLSetupStrategyCertStoreProvider _fixture;
+
+        private readonly Dictionary<string, string> tableNames = new Dictionary<string, string>();
+
+        public SqlBulkCopyTruncation(SQLSetupStrategyCertStoreProvider fixture)
+        {
+            this._fixture = fixture;
+            this.tableNames = fixture.sqlBulkTruncationTableNames;
+        }
+
+        [ConditionalTheory(typeof(DataTestUtility), nameof(DataTestUtility.IsEnclaveEnabledAndConnectionStringsProvided))]
```

Comment:
Why are these tests run for EnclaveEnabled only?

Reply:
The tables are all encrypted by "RANDOMIZED" encrypotion type. There are tests to check failure  when enclave is enabled on the source and not enalbed on the target.

Reply:
fixed to work on AE